### PR TITLE
org.eclipse.tips.core.TipTest.testEqualsObject() fails #140

### PR DESCRIPTION
**Pull request by vrubezhny**
Fixes: #140

### CODE REVIEW

**Comment by jukzi** [comment]
Can you please investigate and fix why this unit test is not executed
https://ci.eclipse.org/platform/job/eclipse.platform.ua/job/PR-142/1/testReport/

**Comment by jukzi** [comment]
Then they are probably not added to a suite or the suite is not referenced in the build configuration.

File: org.eclipse.tips.tests/src/org/eclipse/tips/core/TipTest.java [hunk 1]
```diff
@@ -104,8 +104,8 @@ public void testGetSubject() {
 	public void testEqualsObject() {
 		TestTip testTip = new TestTip(fProvider.getID(), HTML, SUBJECT_TIP);
 		TestTip testTipx = new TestTip(fProvider.getID(), HTML, SUBJECT_TIP);
-		assertNotNull(testTip);
-		assertEquals(testTip, testTip.equals(testTip));
+		assertFalse(testTip.equals(fTip));
+		assertTrue(testTip.equals(testTipx));
```
Comment (jukzi) [inline org.eclipse.tips.tests/src/org/eclipse/tips/core/TipTest.java:108]: better use assertEquals/NotEquals with the right arguments

Comment (akurtakov) [reply org.eclipse.tips.tests/src/org/eclipse/tips/core/TipTest.java:108]: This is in a test specifically for "equals" method thus I believe it's better to call the method directly.

Comment (vrubezhny) [reply org.eclipse.tips.tests/src/org/eclipse/tips/core/TipTest.java:108]: Invocation of `.equals()` is made intentionally - that's what the test is exactly developed for - to test `equals()` work on these objects, as far as I understand.

The duplicating extra testing work done in the following two lines:
```
assertNotEquals(fTip, testTip);
assertEquals(testTipx, testTip);
```
but this is not the goal, it's done more to prove the `.equals()` -like assertions in two previous lines.

**Comment by vrubezhny** [comment]
> Then they are probably not added to a suite or the suite is not referenced in the build configuration.

Exactly

**Pull request merged**